components/HomeScreen.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from 'react';
+
+import LottieLoader from './LottieLoader';
+import Cinema from '../assets/cinema.jpg';
+import { API_KEY } from '../api.js';
+import {
+  HomeMainContainer,
+  ButtonMovie,
+  ButtonMovieText,
+} from '../styled-components/styles';
+
+const MOVIE_URL = `https://api.themoviedb.org/3/movie/top_rated?api_key=${API_KEY}&language=en-US&page=1`;
+
+const HomeScreen = ({ navigation }) => {
+  const [randomMovie, setRandomMovie] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const fetchRandomMovie = () => {
+    fetch(MOVIE_URL)
+      .then(result => {
+        if (result.ok) {
+          return result.json();
+        } else {
+          throw new Error('404');
+        }
+      })
+      .then(json => {
+        setRandomMovie(
+          json.results[Math.floor(Math.random() * json.results.length)]
+        );
+        setIsLoading(false);
```

Comment:
Personally, I would initialise `isLoading` property as `false`, later on set it to `true` at the beginning of your `fetchRandomMovie()` function and at the very end do what you did on this line (31) with setting it to `false` once again